api/utils.ts

```diff
@@ -0,0 +1,28 @@
+import { ToastMessage } from "components/toast"
+
+export interface genericError {
+  message: string
+  request: { status: number }
+}
+
+export const errorHandler = (error: genericError) => {
+  switch (error.request.status) {
+    case 0:
```

Comment:
put `ToastMessage({ type: "error", message: message })` outside of the case,

only put the `message` in case

Reply:
good idea. copy that

Reply:
is it done like this?
![image](https://user-images.githubusercontent.com/91857581/190330223-119fcadc-138c-463f-9299-8ad4e594176d.png)

Reply:
Yeah